float32 x
float32 y
float32 z

int ingredient_type

---
geometry_msgs/Transform end_pose
---